pkg/products/rhsso/reconciler.go

```diff
@@ -366,35 +572,39 @@ func (r *Reconciler) setupOpenshiftIDP(ctx context.Context, installation *integr
 
 	err := serverClient.Get(ctx, k8sclient.ObjectKey{Name: oauthClientSecrets.Name, Namespace: r.ConfigManager.GetOperatorNamespace()}, oauthClientSecrets)
 	if err != nil {
-		return fmt.Errorf("Could not find %s Secret: %w", oauthClientSecrets.Name, err)
+		return errors.Wrapf(err, "Could not find %s Secret", oauthClientSecrets.Name)
 	}
 
 	clientSecretBytes, ok := oauthClientSecrets.Data[string(r.Config.GetProductName())]
 	if !ok {
-		return fmt.Errorf("Could not find %s key in %s Secret", r.Config.GetProductName(), oauthClientSecrets.Name)
+		return errors.Wrapf(err, "Could not find %s key in %s Secret", string(r.Config.GetProductName()), oauthClientSecrets.Name)
 	}
 	clientSecret := string(clientSecretBytes)
 
-	oauthc := &oauthv1.OAuthClient{
+	redirectUris := []string{
+		r.Config.GetHost() + "/auth/realms/openshift/broker/openshift-v4/endpoint",
+	}
+
+	oauthClient := &oauthv1.OAuthClient{
 		ObjectMeta: metav1.ObjectMeta{
 			Name: r.getOAuthClientName(),
 		},
-		Secret: clientSecret,
-		RedirectURIs: []string{
-			r.Config.GetHost() + "/auth/realms/openshift/broker/openshift-v4/endpoint",
-		},
-		GrantMethod: oauthv1.GrantHandlerPrompt,
+		Secret:       clientSecret,
+		RedirectURIs: redirectUris,
+		GrantMethod:  oauthv1.GrantHandlerPrompt,
 	}
 
-	_, err = r.ReconcileOauthClient(ctx, installation, oauthc, serverClient)
+	_, err = r.ReconcileOauthClient(ctx, inst, oauthClient, serverClient)
```

Comment:
We refer to the installation object as `installation` everywhere now, to try to keep our code consistent

Reply:
will update